src/test/scala/it/pagopa/interop/backendforfrontend/AuthorizationApiServiceSpec.scala

```diff
@@ -134,19 +134,19 @@ class AuthorizationApiServiceSpec extends AnyWordSpecLike with SpecHelper with S
         .returns(Success(jwtClaimsSet))
 
       (mockTenantManagement
-        .getBySelfcareId(_: UUID)(_: Seq[(String, String)]))
+        .getBySelfcareId(_: String)(_: Seq[(String, String)]))
         .expects(selfcareId, *)
         .once()
         .returns(Future.failed(ApiError[Problem](404, "oh no!", None, new Exception, Map.empty)))
 
       (mockPartyProcess
-        .getInstitution(_: UUID)(_: Seq[(String, String)], _: ExecutionContext))
+        .getInstitution(_: String)(_: Seq[(String, String)], _: ExecutionContext))
         .expects(selfcareId, *, *)
         .once()
         .returns(
           Future.successful(
             Institution(
-              id = selfcareId,
+              id = UUID.fromString(selfcareId),
```

Comment:
how is it possible this is not breaking? 🤔

Reply:
We never released the interop-party-whatever project with the signature update probably

Reply:
sorry I didn't explain the point properly: as far as I see, `selfcareId` is a string not formatted as UUID, so should't `UUID.fromString` throw an exception when running the test?

Reply:
Well, it breaks
```
- should succeed even if the tenant needs to be upserted *** FAILED ***
[info]   java.lang.IllegalArgumentException: Invalid UUID string: stringThatWillMakeGalalaesHappy
[info]   at java.base/java.util.UUID.fromString(UUID.java:215)
[info]   at it.pagopa.interop.backendforfrontend.AuthorizationApiServiceSpec.$anonfun$new$4(AuthorizationApiServiceSpec.scala:149)```